test/fixtures/commits_after_review.json

```diff
@@ -0,0 +1,56 @@
+{
+  "singleCommit": {
+      "commit": [{
+        "commit": {
+          "committedDate": "2017-10-25T11:27:02Z",
+          "oid": "6c0945cbeea2cbbc97d13a3d9e3fe68bd145b985",
+          "bodyText": "src: fix issue with es-modules",
+          "message": "single commit was pushed after review",
+          "author": {
+            "login": "bar"
+          }
+        }
+    }],
+    "review": [{
+      "bodyText": "Good idea",
+      "state": "APPROVED",
+      "author": {
+        "login": "foo"
+      },
+      "url": "https://github.com/nodejs/node/pull/16438#pullrequestreview-71480624",
+      "publishedAt": "2017-10-24T11:19:25Z"
+    }]
+  },
+  "multipleCommits": {
+    "commits": [{
+      "commit": {
+        "committedDate": "2017-10-25T11:27:02Z",
+        "oid": "f230e691459f8b0f448e1eec1b83fbf7f708eba3",
+        "bodyText": "src: add requested feature",
+        "message": "src: add requested feature\nkjlks jskld ksa",
+        "author": {
+          "login": "bar"
+        }
+      }
+    },{
+      "commit": {
+        "committedDate": "2017-10-25T12:42:02Z",
+        "oid": "9416475a6dc1b27c3e343dcbc07674a439c88db1",
+        "bodyText": "nit: edit mistakes",
+        "message": "nit: fix errors\n fixed requested errors",
+        "author": {
+          "login": "bar"
+        }
+      }
+    }],
+    "review": [{
+      "bodyText": "Good idea",
+      "state": "APPROVED",
+      "author": {
+        "login": "foo"
+      },
+      "url": "https://github.com/nodejs/node/pull/16438#pullrequestreview-89923489",
+      "publishedAt": "2017-09-23T11:19:25Z"
+    }]
+  }
+}
```

Comment:
Could you add a new line here? (We need a lint rule for this, I know.)

Reply:
@apapirovski i don't think so for `json`, every other `json` does not have `EOL`. But i added it either way as it looks better.

Reply:
That's something we'll need to fix. On unix systems a line is only complete if it ends with a new line.